test/katex-spec.js

```diff
@@ -3225,6 +3262,24 @@ describe("A macro expander", function() {
         expect(macros["\\foo"]).toBeTruthy();
     });
 
+    it("\\let copies the definition", () => {
+        expect`\let\foo=\frac\def\frac{}\foo12`.toParseLike`\frac12`;
+        expect`\def\foo{1}\let\bar\foo\def\foo{2}\bar`.toParseLike`1`;
+        expect`\let\foo=\kern\edef\bar{\foo1em}\let\kern=\relax\bar`.toParseLike`\kern1em`;
+        expect`\let\foo{\frac\foo1}{2}`.toParseLike`\frac{1}{2}`;
```

Comment:
I didn't realize the `=` was optional when using `\let`.  In the first one it looks like `\frac` is being redefined, but the result doesn't include this redefinition.  I think it would be useful to have comments for some of these tests to help people that aren't as familiar with the interections between `let` and `def`.

Reply:
I'll add comments on test cases.